addons/web/views/report_templates.xml

```diff
@@ -115,6 +122,9 @@
                         }
                     </script>
                 </t>
+                <t t-foreach="custom_styles" t-as="style">
+                    <t t-raw="style"/>
+                </t>
```

Comment:
What is it about?
The asset manager should already handle the `<style>` tags.

Reply:
It's the whole point of the PR really: adding a custom `<style>` directly in the report layout in order to implement per-company colors/font. It can't be done with regular assets because they're not company-specific, and we don't want to invalidate assets whenever a company changes color I think.
So the trade-off here is to quickly generate a dynamic `<style>` on-the-fly while rendering the report: it's enough for the few CSS rules we need, and it will always be up-to-date.

Or is there something else that's unclear?

Reply:
Do companies change colors that much? I don't think regenerating 1 asset is a big deal.

The problem is that extracting all the `<style>` tags at the end break the CSS inheritance.